app/views/bundlesLanding.scala.html

```diff
@@ -0,0 +1,11 @@
+@styles = {
```

Comment:
Yes, that would also be a good idea. We just need some way of individually specifying the javascript for each page, and providing a root element (with corresponding id) for React to render to.